examples/chip-tool/commands/payload/AttestationResponseParser.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "../clusters/DataModelLogger.h"
+#include "../common/Command.h"
+#include <setup_payload/SetupPayload.h>
+
+class AttestationResponseParser : public Command, public DataModelLogger
+{
+public:
+    AttestationResponseParser() : Command("attestation-response-parse") { AddArgument("payload", &mAttResponse); }
```

Comment:
I'm a little confused by one thing: where is the input to this going to come from?

Reply:
> It is being passed in from the user from the CLI.

Yes, but where is the user getting the thing that they are passing on the CLI?

Reply:
Answered offline.